end-to-end-tests/src/test/java/tech/pegasys/peeps/consensus/CliqueConsensusTest.java

```diff
@@ -13,30 +13,32 @@
 package tech.pegasys.peeps.consensus;
 
 import tech.pegasys.peeps.NetworkTest;
-import tech.pegasys.peeps.NodeConfiguration;
 import tech.pegasys.peeps.SignerConfiguration;
 import tech.pegasys.peeps.network.ConsensusMechanism;
 import tech.pegasys.peeps.network.Network;
 import tech.pegasys.peeps.node.Account;
+import tech.pegasys.peeps.node.Web3Provider;
+import tech.pegasys.peeps.node.Web3ProviderType;
 import tech.pegasys.peeps.node.model.Hash;
 import tech.pegasys.peeps.node.verification.ValueReceived;
 import tech.pegasys.peeps.node.verification.ValueSent;
 
+import org.apache.tuweni.crypto.SECP256K1.KeyPair;
 import org.apache.tuweni.eth.Address;
 import org.apache.tuweni.units.ethereum.Wei;
 import org.junit.jupiter.api.Test;
 
 public class CliqueConsensusTest extends NetworkTest {
 
-  private final NodeConfiguration node = NodeConfiguration.ALPHA;
+  private Web3Provider alphaNode;
   private final SignerConfiguration signer = SignerConfiguration.ALPHA;
 
   @Override
   protected void setUpNetwork(final Network network) {
-    network.addNode(node.id(), node.keys());
-    network.addNode(NodeConfiguration.BETA.id(), NodeConfiguration.BETA.keys());
-    network.set(ConsensusMechanism.CLIQUE, node.id());
-    network.addSigner(signer.id(), signer.resources(), node.id());
+    alphaNode = network.addNode("alpha", KeyPair.random(), Web3ProviderType.GOQUORUM);
```

Comment:
nit: move `alpha` string to a constant?

Reply:
same for `beta` ?

Reply:
its not a constant - its just a name, so prefer not to - it turns up in logs files, and is an arbitrary value